pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -29,32 +29,34 @@ import (
 
 // ValidateDefaultPreemptionArgs validates that DefaultPreemptionArgs are correct.
 func ValidateDefaultPreemptionArgs(args config.DefaultPreemptionArgs) error {
-	if err := validateMinCandidateNodesPercentage(args.MinCandidateNodesPercentage); err != nil {
-		return err
+	var allErrs field.ErrorList
+	p := field.NewPath("minCandidateNodesPercentage")
+	if err := validateMinCandidateNodesPercentage(args.MinCandidateNodesPercentage, p); err != nil {
+		allErrs = append(allErrs, err)
 	}
-	if err := validateMinCandidateNodesAbsolute(args.MinCandidateNodesAbsolute); err != nil {
-		return err
+	if err := validateMinCandidateNodesAbsolute(args.MinCandidateNodesAbsolute, field.NewPath("minCandidateNodesAbsolute")); err != nil {
+		allErrs = append(allErrs, err)
 	}
 	if args.MinCandidateNodesPercentage == 0 && args.MinCandidateNodesAbsolute == 0 {
-		return fmt.Errorf("both minCandidateNodesPercentage and minCandidateNodesAbsolute cannot be zero")
+		allErrs = append(allErrs, field.Invalid(p, args.MinCandidateNodesPercentage, "cannot be zero at the same time as minCandidateNodesAbsolute"))
```

Comment:
Add 2 errors, one for each field.

Reply:
Like this?
```go
allErrs = append(allErrs, field.Invalid(field.NewPath("minCandidateNodesPercentage"), args.MinCandidateNodesPercentage, "cannot be zero at the same time as minCandidateNodesAbsolute"))
allErrs = append(allErrs, field.Invalid(field.NewPath("minCandidateNodesAbsolute"), args.MinCandidateNodesAbsolute, "cannot be zero at the same time as minCandidateNodesPercentage"))
```

Reply:
Yes, but you can do that in a single `append`

Reply:
Perhaps it should be repeated twice semantically. But I think this is a bit redundant.